e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    before(() => {
+        // # Login as sysadmin and visit customization system console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/customization');
+    });
+
+    it('SC20336 - Can change Custom Brand Image setting', () => {
+        // # Set Enable Custom Branding to true to be able to upload custom brand image
+        cy.get('[id="TeamSettings.EnableCustomBrandtrue"]').check();
+
+        // * Verify that setting is visible and matches text content
+        cy.get('.remove-image').closest('.form-group').find('label').should('be.visible').and('have.text', 'Custom Brand Image:');
+
+        // * Verify that help setting is visible and matches text content
+        const contents = ['Customize your user experience by adding a custom image to your login screen. Recommended maximum image size is less than 2 MB.'];
+        cy.get('.remove-image').closest('.form-group').find('.help-text').should('be.visible').and('have.text', contents[0]);
+
+        // # upload the image
+        cy.fileUpload('input[type="file"]');
```

Comment:
Please add `data-testid={this.props.id}` to [BrandImageSetting label](https://github.com/mattermost/mattermost-webapp/blob/38fab4776ae8faaf51f5200a5d000c5cb0133bf7/components/admin_console/brand_image_setting/brand_image_setting.jsx#L208), so above lines can easily get by and wrap children as:
```javascript
        cy.findByTestId('CustomBrandImage').should('be.visible').within(() => {
            // * Verify that setting is visible and matches text content
            cy.get('label').should('be.visible').and('have.text', 'Custom Brand Image:');

            // * Verify that help setting is visible and matches text content
            const contents = 'Customize your user experience by adding a custom image to your login screen. Recommended maximum image size is less than 2 MB.';
            cy.get('.help-text').should('be.visible').and('have.text', contents);

            // # upload the image
            cy.fileUpload('input');
        });
```